chainio/clients/avsregistry/writer.go

```diff
@@ -32,13 +33,9 @@ type AvsRegistryWriter interface {
 	// socket? RegisterOperatorInQuorumWithAVSRegistryCoordinator is used to register a single operator with the AVS's
 	// registry coordinator. - operatorEcdsaPrivateKey is the operator's ecdsa private key (used to sign a message to
 	// register operator in eigenlayer's delegation manager)
-	//  - operatorToAvsRegistrationSigSalt is a random salt used to prevent replay attacks
-	//  - operatorToAvsRegistrationSigExpiry is the expiry time of the signature
 	RegisterOperatorInQuorumWithAVSRegistryCoordinator(
```

Comment:
should be not have a breaking change add another method and keep it deprecated or keep both fn?? I know you would want to keep the function name but think we should avoid breaking ?

Reply:
What name would you use?